app/components/Game/Game.js

```diff
@@ -16,34 +19,71 @@ export default function Game({
     endButtonPressed,
     goButtonPressed,
   },
-  handleClicks,
+  handleClicks: {
+    handleClickOnBoard,
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+  },
 }) {
-  const gameHeaderState = {
+  const instructionsState = {
     pathStartingPoint,
     pathEndingPoint,
     startButtonPressed,
     endButtonPressed,
   };
-  const gameScreenState = {
+  const instructionsProps = { instructionsState, };
+  const instructions = (
+    <div className="instructionsWrapper" key="2">
+      <GameInstructions { ...instructionsProps } />
+    </div>
+  );
+  const settingsText = '<-- Settings';
+  const linkProps = {
+    className: 'settingsLink',
+    to: { pathname: '/settings', },
+    children: settingsText,
+    key: 1,
+  };
+  const settingsLink = <Link { ...linkProps } />;
+  const gameHeaderProps = { children: [ settingsLink, instructions ], };
+  const buttonsState = {
+    pathStartingPoint,
+    pathEndingPoint,
+    startButtonPressed,
+    endButtonPressed,
+    goButtonPressed,
+  };
+  const boardState = {
     colorA,
     colorB,
     colorC,
     board,
     pathStartingPoint,
     pathEndingPoint,
-    startButtonPressed,
-    endButtonPressed,
-    goButtonPressed,
   };
-  const gameScreenProps = {
-    gameScreenState,
-    handleClicks,
+  const buttonPanelProps = {
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+    buttonsState,
+    key: 2,
+  };
+  const boardProps = {
+    handleClickOnBoard,
+    boardState,
   };
-  const gameHeaderProps = { gameHeaderState, };
+  const BoardContainer = () => (
+    <div className="boardContainer" key="1">
+      <Board { ...boardProps } />
+    </div>
+  );
+  const buttonsPanel = <GameButtonsPanel { ...buttonPanelProps } />;
```

Comment:
B

Reply:
done

although it will move to the JSX as a whole soon enough..